### DISCUSSION TITLE
STM32 IDE with autoconf file

### DISCUSSION THREAD

**Discussion by yuthnea** [bitcraze/discussions#1545] [General]
I am go through Crazyflie-Firmware and I try to find out where is autoconf.h. The question is file autoconf.h is file auto-generate or file created. does i could use this file in STMcube IDE.

**Comment by ataffanel** [comment]
This file is indeed auto generated by the kbuild build system. Once you compile the firmware, the file will be in ```./build/include/generated/autoconf.h```. You are not supposed to touch this file, it will be generated at each build from the build configuration.
